lib/bolt/plugin/prompt.rb

```diff
@@ -19,9 +19,9 @@ def validate_resolve_reference(opts)
       end
 
       def resolve_reference(opts)
-        STDOUT.print "#{opts['message']}:"
+        STDERR.print "#{opts['message']}:"
```

Comment:
Should we only print to STDERR when `future` is set?

Reply:
Do we have access to the config from plugins?

Reply:
From the ruby ones for sure.

Reply:
Updated.